eng/CodeAnalysis.test.globalconfig

```diff
@@ -792,7 +795,7 @@ dotnet_diagnostic.CA5383.severity = none
 # CA5384: Do Not Use Digital Signature Algorithm (DSA)
 dotnet_diagnostic.CA5384.severity = none
 
-# CA5385: Use Rivest�Shamir�Adleman (RSA) Algorithm With Sufficient Key Size
+# CA5385: Use Rivest–Shamir–Adleman (RSA) Algorithm With Sufficient Key Size
```

Comment:
```suggestion
# CA5385: Use Rivest-Shamir-Adleman (RSA) Algorithm With Sufficient Key Size
```
This doesn't look right (Unicode em- or en-dash causing encoding havoc?) -- replace with plain dashes.

Reply:
Thanks:
https://github.com/dotnet/roslyn-analyzers/pull/5985